CI Failing on master: test_interactive `test_complete_symbol_nonmodule_1`

I discovered this while working on #257 after endless debugging figured about it was failing on master.

Failing CI: https://github.com/deshaw/pyflyby/actions/runs/5695216377/job/15676988960
Passing same CI (about a week ago): https://github.com/deshaw/pyflyby/actions/runs/5695216377/attempts/1

Traceback (diff):

```python
E       Failed: Expected:
E            b'In [1]: import pyflyby; pyflyby.enable_auto_importer()'
E            b'[PYFLYBY] import gravesend60063393'
E            b'in the river'
E            b'In [2]: print(gravesend60063393.river)'
E            b'in the river'
E            b'in the river'
E            b'Medway'
E            b'... the island'
E            b'In [3]: print(gravesend60063393.island)'
E            b'on the island'
E            b'on the island'
E            b'Canvey'
E       Result:
E            b'In [1]: import pyflyby; pyflyby.enable_auto_importer()'
E            b'[PYFLYBY] import gravesend60063393'
E            b'In [2]: print(gravesend60063393.river)'
E            b'in the river'
E            b'in the river'
E            b'Medway'
E            b'In [3]: print(gravesend60063393.is\x1bisland)'
E            b'on the island'
E            b'on the island'
E            b'Canvey'
E       Diff:
E            In [1]: import pyflyby; pyflyby.enable_auto_importer()
E            [PYFLYBY] import gravesend60063393
E          - in the river
E            In [2]: print(gravesend60063393.river)
E            in the river
E            in the river
E            Medway
E          - ... the island
E          - In [3]: print(gravesend60063393.island)
E          + In [3]: print(gravesend60063393.isisland)
E          ?                                   +++
E       
E            on the island
E            on the island
E            Canvey
E       Diff Repr:
E          '  In [1]: import pyflyby; pyflyby.enable_auto_importer()'
E          '  [PYFLYBY] import gravesend60063393'
E          '- in the river'
E          '  In [2]: print(gravesend60063393.river)'
E          '  in the river'
E          '  in the river'
E          '  Medway'
E          '- ... the island'
E          '- In [3]: print(gravesend60063393.island)'
E          '+ In [3]: print(gravesend60063393.is\x1bisland)'
E          '?                                   +++\n'
E          '  on the island'
E          '  on the island'
E          '  Canvey'

tests/test_interactive.py:2488: Failed
```

in `is\x1bis` `\x1b` is an escape sequence for the terminal (also known as `\033`), this point toward a terminal thing either in IPython or likely prompt toolkit. 

That does not explain the following though.

```
E            [PYFLYBY] import gravesend60063393
E          - in the river
E            In [2]: print(gravesend60063393.river)
E            in the river
```

--- a/tests/test_interactive.py
+++ b/tests/test_interactive.py
@@ -2488,14 +2488,11 @@ def test_complete_symbol_nonmodule_1(frontend, tmp):
     ipython(
             """
             In [1]: import pyflyby; pyflyby.enable_auto_importer()
-            In [2]: print(gravesend60063\t393.r\t
-            [PYFLYBY] import gravesend60063393{}
-            In [2]: print(gravesend60063393.river)
-            in the river
+            [PYFLYBY] import gravesend60063393
+            In [2]: print(gravesend60063\t393.r\tiver){}
             in the river
             Medway
-            ... the island
-            In [3]: print(gravesend600633\t93.is\tland)
+            In [3]: print(gravesend600633\t93.isl\tand)
             on the island
             on the island
             Canvey

--- a/tests/test_interactive.py
+++ b/tests/test_interactive.py
@@ -2452,7 +2452,10 @@ def test_complete_symbol_bad_as_1(frontend, tmp):
     )
 
 
-
+@pytest.mark.skipif(
+    sys.version_info == (3, 7),
+    reason="We're dropping support for 3, 7 anyways",
+)
 @retry
 def test_complete_symbol_nonmodule_1(frontend, tmp):
     # Verify that completion works even if a module replaced itself in

--- a/tests/test_interactive.py
+++ b/tests/test_interactive.py
@@ -2453,7 +2453,7 @@ def test_complete_symbol_bad_as_1(frontend, tmp):
 
 
 @pytest.mark.skipif(
-    sys.version_info == (3, 7),
+    sys.version_info < (3, 8),
     reason="We're dropping support for 3, 7 anyways",
 )
 @retry

